tests/sentry/snuba/metrics/test_query_builder.py

```diff
@@ -1628,7 +1669,15 @@ def test_resolve_tags_with_match_and_filterable_tag(self):
         )
 
     def test_resolve_tags_with_match_and_deep_filterable_tag(self):
-        indexer.record(use_case_id=self.use_case_id, org_id=self.org_id, string="environment")
+        indexer.record(
+            use_case_id=(
+                UseCaseID.SESSIONS
+                if self.use_case_id is UseCaseKey.RELEASE_HEALTH
+                else UseCaseID.TRANSACTIONS
```

Comment:
can't all these repeated conditionals come from the mapping that you added in use_case_id_registry?

Reply:
fixed now, thanks